Content.Server/Xenoarchaeology/Artifact/AddEdgeBetweenArtifactNodesCommand.cs

```diff
@@ -0,0 +1,126 @@
+using Content.Server.Administration;
+using Content.Shared.Administration;
+using Content.Shared.Xenoarchaeology.Artifact.Components;
+using Robust.Shared.Console;
+
+namespace Content.Server.Xenoarchaeology.Artifact;
+
+[AdminCommand(AdminFlags.Debug)]
+public sealed class AddEdgeBetweenArtifactNodesCommand : LocalizedEntityCommands
```

Comment:
```suggestion
public sealed class AddEdgeBetweenNodesCommand : LocalizedEntityCommands
```
keep consistent with command name